server/http/mappings/http.go

```diff
@@ -112,28 +112,28 @@ func (m Model) HTTPResponse(in openapi.HttpResponse) *model.HTTPResponse {
 }
 
 func (m Model) Auth(in openapi.HttpAuth) *model.HTTPAuthenticator {
-	var props map[string]string
-
-	switch in.Type {
-	case "apiKey":
-		props = map[string]string{
-			"key":   in.ApiKey.Key,
-			"value": in.ApiKey.Value,
-			"in":    in.ApiKey.In,
-		}
-	case "basic":
-		props = map[string]string{
-			"username": in.Basic.Username,
-			"password": in.Basic.Password,
-		}
-	case "bearer":
-		props = map[string]string{
-			"token": in.Bearer.Token,
-		}
-	}
+	// var props map[string]string
+
+	// switch in.Type {
+	// case "apiKey":
+	// 	props = map[string]string{
+	// 		"key":   in.ApiKey.Key,
+	// 		"value": in.ApiKey.Value,
+	// 		"in":    in.ApiKey.In,
+	// 	}
+	// case "basic":
+	// 	props = map[string]string{
+	// 		"username": in.Basic.Username,
+	// 		"password": in.Basic.Password,
+	// 	}
+	// case "bearer":
+	// 	props = map[string]string{
+	// 		"token": in.Bearer.Token,
+	// 	}
+	// }
 
 	return &model.HTTPAuthenticator{
-		Type:  in.Type,
-		Props: props,
+		Type: in.Type,
+		// Props: props,
```

Comment:
Don't you have to map the arguments for each auth type?

Reply:
uppps, forgot to fix this. Updated